HIP/hip-584.md

```diff
@@ -49,7 +49,7 @@ Information will be made available via new REST API
 #### Read-only queries and gas estimate REST API
 
 A new  `/api/v1/contracts/call` POST REST API will be added to execute read-only queries and gas estimate.
-Query parameter `estimate`(default = false) would determine what the result represents.
```

Comment:
I do still think there will be a default of false even if it's moved into the body and that should be mentioned here.